app/components/properties-panel/properties-panel.component.html

```diff
@@ -55,7 +55,13 @@
 
                             <template [ngIf]="pair[1].subtype(Type.Node)">
                                 <label  class="node-label">
-                                    {{selectedEntity[group[1]].get(pair[0]).drawableProperties.get('Label')}}
+                                    <!-- TODO: Figure out a better way to allow nodes to sometimes be null
+                                        
+                                        Specifically, this doesn't have a value yet, and this breaks a lot of 
+                                        promises that say that there is always a start state, that is, we can't 
```

Comment:
In the case of DFAs, we should guarantee that there is always a start state before we send it to the plugin if the plugin doesn't defines start state as optional. So sinap should be able to mark that there isn't one yet, so the graph isn't valid. The type constraints apply to graphs being sent to the plugin.